pkg/pinger/pinger.go

```diff
@@ -46,24 +63,26 @@ var (
 	// Even though we set up the pinger to run continuously, we still have to give it a non-zero timeout else it will
 	// fail so set a really long one.
 	defaultPingTimeout = 87600 * time.Hour
+
+	logger = log.Logger{Logger: logf.Log.WithName("Pinger")}
 )
 
-type PingerInterface interface {
+type Interface interface {
 	Start()
 	Stop()
 	GetLatencyInfo() *LatencyInfo
 	GetIP() string
 }
 
-type PingerConfig struct {
+type Config struct {
 	IP                 string
 	Interval           time.Duration
 	Timeout            time.Duration
 	MaxPacketLossCount uint
 }
 
-type pingerInfo struct {
-	sync.Mutex
+type Info struct {
```

Comment:
This is the private implementation and shouldn't be exported.

```suggestion
type pingerImpl struct {
```